gammapy/irf/effective_area.py

```diff
@@ -570,3 +563,15 @@ def peek(self, figsize=(15, 5)):
         self.plot_energy_dependence(ax=axes[0])
         self.plot_offset_dependence(ax=axes[1])
         plt.tight_layout()
+
+    def to_table(self):
+        table = Table()
+        table['ENERG_LO'] = [self.data.axis('energy').lo]*self.data.axis('energy').unit
```

Comment:
You currently have this code to insert a new length 1 dimension, to fit the array into a single FITS table row as in the format that we use:
```
table['ENERG_LO'] = [self.data.axis('energy').lo]*self.data.axis('energy').unit
```
I would suggest to write it in this style instead everywhere:
```
table['ENERG_LO'] = self.data.axis('energy').lo[np.newaxis]
```
The result is the same, I just find the code a little shorter and simpler.

In the `from_table` methods I think we should use this to again drop this "fit into single row" dimension that is a curiosity of the FITS BINTABLE serialisation format:
```
energy_lo = table['ENERG_LO'].quantity[0]
```
Then the code is pretty symmetrical and easy to read.

@thomasarmstrong - Does that sound OK to you?

If yes, would you mind also fixing up these lines to use that coding pattern?
https://gist.github.com/cdeil/40c787d6188bee1c2f214246a84d3ca0
(previously we used `squeeze` to remove the extra first length 1 dimension. But that will actually also squeeze out other axes if they have length 1, which can happen, so really that is a bug waiting to happen)